CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 4.2.0
+
+- Add canMakePayments method (https://github.com/RevenueCat/react-native-purchases/pull/244)
+- Add PR template (https://github.com/RevenueCat/react-native-purchases/pull/263)
+
```

Comment:
The VERSIONS.md also needs to be updated

Reply:
🤦 forgot to save changes in VS